config/prrte_setup_hwloc.m4

```diff
@@ -34,7 +36,7 @@ AC_DEFUN([PRRTE_HWLOC_CONFIG],[
               [PRRTE_HWLOC_HEADER="<hwloc.h>"],
               [PRRTE_HWLOC_HEADER="$with_hwloc_header"
                prrte_hwloc_header_given=1])
-        prrte_hwloc_support=1
+        prrte_hwloc_support=0
```

Comment:
Doesn't this indicate that PRRTE will not include hwloc support? Maybe I'm misunderstanding the impact here - I wouldn't want us to lose the ability to bind processes.

Reply:
Reading your comment more closely, it seems to me that the correct fix would be to add a "without hwloc" clause to this ```if``` chain. As written, the PR would say that if the header is given, then disable hwloc support in PRRTE. Let me try the alternative approach.

Reply:
After reading thru the OMPI config for hwloc, I confess I am even more confused here. According to that config, if the external hwloc isn't available of is too old, then we must use the embedded version. It also explicitly states that OMPI requires hwloc and will not build without a suitable version. So I don't understand why setting this flag telling PRRTE not to use hwloc is the right answer. Can you provide further explanation?

Reply:
Yep, that was not the right fix.

We should really move the block below one line up

```
    if test $prrte_hwloc_support = "1"; then
        AC_MSG_CHECKING([if external hwloc version is 1.5 or greater])
        AC_COMPILE_IFELSE(
              [AC_LANG_PROGRAM([[#include <hwloc.h>]],
              [[
    #if HWLOC_API_VERSION < 0x00010500
    #error "hwloc API version is less than 0x00010500"
    #endif
              ]])],
              [AC_MSG_RESULT([yes])],
              [AC_MSG_RESULT([no])
               AC_MSG_ERROR([Cannot continue])])
    fi
```

Reply:
kewl - can you update?